cot/src/form.rs

```diff
@@ -212,6 +212,12 @@ impl From<email_address::Error> for FormFieldValidationError {
     }
 }
 
+impl From<url::ParseError> for FormFieldValidationError {
```

Comment:
Similarly to my concerns about publicly exposed `url::ParseError` above, I think we should always manually construct `FormFieldValidationError` from our custom struct/`ParseError` instead of providing a `From` impl.